pkg/configuration/configuration.go

```diff
@@ -46,6 +46,23 @@ const (
 
 	// NotificationDeliveryServiceMailgun is the notification delivery service to use during production
 	NotificationDeliveryServiceMailgun = "mailgun"
+
+	varNotificationDeliveryService = "notification.delivery.service"
+
+	// varDurationBeforeNotificationDeletion specifies the duration before a notification will be deleted
+	varDurationBeforeNotificationDeletion = "duration.before.notification.deletion"
+
+	// varMailgunDomain specifies the host operator mailgun domain
+	varMailgunDomain = "mailgun.domain"
+
+	// varMailgunAPIKey specifies the host operator mailgun api key
+	varMailgunAPIKey = "mailgun.api.key"
+
+	// mailgunDomain defines mailgun domain key
+	mailgunDomain = "mailgun-domain"
+
+	// mailgunAPIKey defines mailgun api key
+	mailgunAPIKey = "mailgun-api-key"
```

Comment:
These similarly-named constants are a little confusing, could we perhaps add some additional explanation to the comments to describe how they are used?

Reply:
hmm, i think these names are pretty clear to me - what should i add to make it more clear? could you provide examples?

Reply:
these are actually removed since they are unused. however i did add it as part of the var description for these fields. see: d1250e5db929e62d1956927d47ae30ef533a9235